pkg/logentry/stages/template.go

```diff
@@ -32,6 +34,10 @@ var (
 		"TrimPrefix": strings.TrimPrefix,
 		"TrimSuffix": strings.TrimSuffix,
 		"TrimSpace":  strings.TrimSpace,
+		"Sha256": func(salt string, s string) string {
```

Comment:
I like this, but it'd be cool if it was multi-arity. For instance, supporting `| Sha256` and `| Sha256 "salt"`.

I can followup this PR with this change myself.